packages/client/src/components/Dropdown/index.scss

```diff
@@ -0,0 +1,104 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:400,700,900');
+
+/* RESETS */
+
+html {
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+*, *:before, *:after {
+  -webkit-box-sizing: inherit;
+  box-sizing: inherit;
+}
+
+body {
```

Comment:
let's also not change global styles here, this can be hard to debug. prefer styled-components so we can scope styles to the component level